### PR TITLE
Brush up select Equipment and Feat effects

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/feat-effects/effect-ceremony-of-protection.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Ceremony of Protection",
     "system": {
         "description": {
-            "value": "<p>You gain a +1 circumstance bonus to AC until the start of your next turn.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Ceremony of Protection]</p>\n<p>You gain a +1 circumstance bonus to AC until the start of your next turn.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-ceremony-of-protection.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Ceremony of Protection]</p>\n<p>You gain a +1 circumstance bonus to AC.</p>"
```

File: packs/feat-effects/effect-born-to-the-trees.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Born to the Trees",
     "system": {
         "description": {
-            "value": "<p>You gain a climb Speed equal to your Speed and a +1 circumstance bonus to Acrobatics checks to @UUID[Compendium.pf2e.actionspf2e.Item.Balance] and Athletics checks to @UUID[Compendium.pf2e.actionspf2e.Item.Long Jump] and @UUID[Compendium.pf2e.actionspf2e.Item.High Jump].</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Born to the Trees]</p>\n<p>You gain a climb Speed equal to your Speed and a +1 circumstance bonus to Acrobatics checks to @UUID[Compendium.pf2e.actionspf2e.Item.Balance] and Athletics checks to @UUID[Compendium.pf2e.actionspf2e.Item.Long Jump] and @UUID[Compendium.pf2e.actionspf2e.Item.High Jump].</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-born-to-the-trees.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Born to the Trees]</p>\n<p>You gain a climb Speed equal to your Speed and a +1 circumstance bonus to Acrobatics checks to Balance and Athletics checks to Long Jump and High Jump.</p>"
```

File: packs/equipment-effects/effect-wand-of-fey-flames.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Wand of Fey Flames",
     "system": {
         "description": {
-            "value": "<p>For the duration of the spell, you can use the limning flames as a source for your magic. Creatures affected by your <em>faerie fire</em> must succeed at a Will save against your spell DC or take a -2 status penalty to their Will saves against enchantment and illusions spells as long as they are affected by <em>faerie fire</em>. You can cast an enchantment or illusion spell on such a creature up to a distance twice the spell's normal range. If the spell has a range of touch, or doesn't have a range, it is unaffected by the increase in range from the <em>faerie fire</em>.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Wand of Fey Flames]</p>\n<p>You take a -2 status penalty to their Will saves against enchantment and illusions spells.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-wand-of-fey-flames.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Wand of Fey Flames]</p>\n<p>You take a -2 status penalty to your Will saves against enchantment and illusions spells.</p>"
```

File: packs/equipment-effects/effect-whispering-staff-ally.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Whispering Staff (Ally)",
     "system": {
         "description": {
-            "value": "<p>Whenever you and your affected allies attempt to Recall Knowledge or attempt a saving throw against a mental effect, you roll twice and take the better result. This is a fortune effect.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Whispering Staff]</p>\n<p>Whenever you attempt to Recall Knowledge or attempt a saving throw against a mental effect, you roll twice and take the better result. This is a fortune effect.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-ally.json:7]: Consider changing the predicate to `"item:trait:mental"`.

File: packs/equipment-effects/effect-whispering-staff-enemy.json [hunk 5]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Whispering Staff (Enemy)",
     "system": {
         "description": {
-            "value": "<p>Through the staff's strange whispering, you gain a glimpse into the mind and desires of one creature you can see within 30 feet. Until the end of your next turn, that creature is @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] to you and takes a –2 circumstance penalty to saving throws against your spells.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Whispering Staff]</p>\n<p>You are take a –2 circumstance penalty to saving throws against spells.</p>"
```
Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-enemy.json:7]: Consider updating the selector to `"selector": "saving-throw"`.

Also, the "your spells" part could be handled with a signature predicate.

File: packs/feat-effects/effect-bone-swarm.json [hunk 6]
```diff
@@ -4,7 +4,7 @@
...
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Bone Swarm]</p>\n<p>You become Huge, gain a fly Speed of 40 feet, have weakness 5 to area and splash damage. You are immune to the @UUID[Compendium.pf2e.conditionitems.Item.Grabbed], @UUID[Compendium.pf2e.conditionitems.Item.Prone], and @UUID[Compendium.pf2e.conditionitems.Item.Restrained] conditions.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-bone-swarm.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Bone Swarm]</p>\n<p>You become Huge, gain the swarm trait, and gain a fly Speed of 40 feet. You are immune to the grabbed, prone, and restrained conditions. You have weakness 5 to area and splash damage.</p>"
```

Comment (simonward) [reply packs/feat-effects/effect-bone-swarm.json:7]: Consider adding the swarm trait:
```
{
  "key": "ActorTraits",
  "add": [
    "swarm"
  ]
}
```

File: packs/feat-effects/effect-brightness-seeker.json [hunk 7]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Brightness Seeker",
     "system": {
         "description": {
-            "value": "<p><strong>Trigger</strong> You attempt an attack roll, skill check, or saving throw while performing the course of action from your @UUID[Compendium.pf2e.spells-srd.Item.Augury], but you haven't rolled yet.</p>\n<hr />\n<p><strong>Effect</strong> You gain a +1 status bonus to the triggering check, or a +2 status bonus if the result of the augury was woe and you proceeded anyway.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.actionspf2e.Item.Call Upon the Brightness]</p>\n<p>You gain a +1 status bonus to the triggering check, or a +2 status bonus if the result of the augury was woe and you proceeded anyway.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-brightness-seeker.json:7]: Consider condensing to a single FlatModifier:

```
{
	"choices": [
		{
			"label": "PF2E.SpecificRule.Elf.BrightnessSeeker.Weal",
			"value": 1
		},
		{
			"label": "PF2E.SpecificRule.Elf.BrightnessSeeker.Woe",
			"value": 2
		},
		{
			"label": "PF2E.SpecificRule.Elf.BrightnessSeeker.WealAndWoe",
			"value": 1
		}
	],
	"flag": "brightnessSeeker",
	"key": "ChoiceSet",
	"prompt": "PF2E.SpecificRule.Elf.BrightnessSeeker.Prompt",
	"rollOption": "brightness-seeker"
},
{
	"key": "FlatModifier",
	"predicate": [
		"call-upon-the-brightness"
	],
	"selector": [
		"attack",
		"saving-throw",
		"skill-check"
	],
	"type": "status",
	"value": "{item|flags.pf2e.rulesSelections.brightnessSeeker}"
}
```

Comment (simonward) [reply packs/feat-effects/effect-brightness-seeker.json:7]: Removed the `rollOption` as it's not needed.

File: packs/equipment-effects/effect-whispering-staff-enemy.json (hunk 5 above)
Comment (ammalagonc) [reply packs/equipment-effects/effect-whispering-staff-enemy.json:7]: Check when you get a moment to see if this works.

Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-enemy.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Whispering Staff]</p>\n<p>You take a –2 circumstance penalty to saving throws against the origin's spells.</p>"
```

Comment (simonward) [reply packs/equipment-effects/effect-whispering-staff-enemy.json]: > Check when you get a moment to see if this works.

It looks reasonable. Did you test?

File: packs/equipment-effects/effect-whispering-staff-enemy.json [hunk 8]
```diff
@@ -25,12 +25,10 @@
                 "key": "FlatModifier",
                 "label": "Whispering Staff",
                 "predicate": [
-                    "origin"
+                    "target:signature:{item|origin.signature}"
```
Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-enemy.json:28]:
```suggestion
                    "self:signature:{item|origin.signature}"
```

Comment (simonward) [reply packs/equipment-effects/effect-whispering-staff-enemy.json:28]: I think this is more correct?

File: packs/equipment-effects/effect-whispering-staff-enemy.json (hunk 5 above)
Comment (ammalagonc) [reply packs/equipment-effects/effect-whispering-staff-enemy.json]: I did. 

Sorry, bad habit from work. I always ask people to double-check as a sanity check.

File: packs/equipment-effects/effect-whispering-staff-enemy.json [hunk 9]
```diff
@@ -25,12 +25,10 @@
                 "key": "FlatModifier",
                 "label": "Whispering Staff",
                 "predicate": [
-                    "origin"
+                    "self:signature:{item|origin.signature}"
```
Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-enemy.json:28]:
```suggestion
                    "origin:signature:{item|origin.signature}"
```

Comment (simonward) [reply packs/equipment-effects/effect-whispering-staff-enemy.json:28]: My mistake. I think this is right.

File: packs/equipment-effects/effect-whispering-staff-enemy.json [hunk 10]
```diff
@@ -25,12 +25,10 @@
                 "key": "FlatModifier",
                 "label": "Whispering Staff",
                 "predicate": [
-                    "origin"
+                    "origin:signature:{item|origin.signature}"
```
Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-enemy.json:28]:
```suggestion
                    "item:type:spell",
                    "origin:signature:{item|origin.signature}"
```

File: packs/equipment-effects/effect-whispering-staff-enemy.json [hunk 11]
```diff
@@ -25,12 +25,11 @@
                 "key": "FlatModifier",
                 "label": "Whispering Staff",
```
Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-enemy.json:26]:
```suggestion
```

File: packs/equipment-effects/effect-whispering-staff-enemy.json [hunk 12]
```diff
@@ -23,14 +23,12 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Whispering Staff",
                 "predicate": [
-                    "origin"
+                    "item:type:spell",
+                    "origin:signature:{item|origin.signature}"
                 ],
                 "selector": [
```
Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-enemy.json:30]:
```suggestion
                "selector": "saving-throw",
```

File: packs/equipment-effects/effect-whispering-staff-enemy.json [hunk 13]
```diff
@@ -23,14 +23,12 @@
...
-                    "fortitude",
-                    "reflex",
-                    "will"
+                    "saving-throw"
```
Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-enemy.json:31]:
```suggestion
```

File: packs/equipment-effects/effect-whispering-staff-enemy.json [hunk 14]
```diff
@@ -23,14 +23,12 @@
...
                 ],
```
Comment (simonward) [inline packs/equipment-effects/effect-whispering-staff-enemy.json:32]:
```suggestion
```

**Pull request merged**